Resources/Prototypes/_DV/Entities/Objects/Misc/pf_sheets.yml

```diff
@@ -0,0 +1,536 @@
+  parent: Paper
+  id: PaperCFSheet # legally gray zone of using "PF2E"
+  description: 'A sheet for your Carpfinder 2E Adventures.'
+  components:
+  - type: Paper
+    contentSize: 10000
+    content: book-cf-sheet
+  - type: Sprite
+    layers:
+    - state: paper
+      color: "#cccccc"
+    - state: paper_words
+      map: ["enum.PaperVisualLayers.Writing"]
+      color: "#cccccc" #aaaaaaaaaaaaaaaaaaaaaaa
+      visible: false
+    - state: paper_stamp-generic
+      map: ["enum.PaperVisualLayers.Stamp"]
+      visible: false
+  - type: PaperVisuals
+    backgroundImagePath: "/Textures/Interface/Paper/paper_background_default.svg.96dpi.png"
+    contentImagePath: "/Textures/Interface/Paper/paper_content_lined.svg.96dpi.png"
+    backgroundModulate: "#cccccc"
+    contentImageModulate: "#cccccc"
+    backgroundPatchMargin: 16.0, 16.0, 16.0, 16.0
+    contentMargin: 16.0, 16.0, 16.0, 16.0
+
+- type: entity
+  id: CFPremadeFolder
+  parent: BoxFolderBase
+  name: CF2E Pregenerated Characters
+  description: For starting your CF2E Adventures quickly!
+  suffix: Forms
+  components:
+  - type: Sprite
+    layers:
+    - state: folder-white
+    - state: folder-base
+  - type: StorageFill
+    contents:
+    - id: PaperCFSheetFighterPremade
+    - id: PaperCFSheetWizardPremade
+    - id: PaperCFSheetBardPremade
+    - id: PaperCFSheetRoguePremade
+    - id: PaperCFSheetClericPremade
+    - id: PaperCFSheetDruidPremade
+    - id: PaperCFSheetDruidFamiliarPremade
+    - id: PaperCFSheetWitchPremade
+    - id: PaperCFSheetRangerPremade
+    - id: PaperCFSheetAlchemistPremade
+    - id: PaperCFSheetBarbarianPremade
+    - id: PaperCFSheetChampionPremade
+    - id: PaperCFSheetInvestigatorPremade
+    - id: PaperCFSheetMonkPremade
+    - id: PaperCFSheetOraclePremade
+    - id: PaperCFSheetSorcererPremade
+    - id: PaperCFSheetSwashbucklerPremade
+  - type: StaticPrice
+    price: 100
+
+- type: entity
+  name: CF fighter premade character sheet
+  parent: Paper
+  id: PaperCFSheetFighterPremade
+  description: 'YOu are an undisputed master of weaponry and martial techniques, capable of deflecting attacks, striking down foes, and controlling the flow of combat.'
+  components:
+  - type: Paper
+    contentSize: 10000
+    content: book-cf-sheet-fighter-premade
+  - type: Sprite
+    layers:
+    - state: paper
+      color: "#cccccc"
+    - state: paper_words
+      map: ["enum.PaperVisualLayers.Writing"]
+      color: "#cccccc" #aaaaaaaaaaaaaaaaaaaaaaa
+      visible: false
+    - state: paper_stamp-generic
+      map: ["enum.PaperVisualLayers.Stamp"]
+      visible: false
+  - type: PaperVisuals
+    backgroundImagePath: "/Textures/Interface/Paper/paper_background_default.svg.96dpi.png"
+    contentImagePath: "/Textures/Interface/Paper/paper_content_lined.svg.96dpi.png"
+    backgroundModulate: "#cccccc"
+    contentImageModulate: "#cccccc"
+    backgroundPatchMargin: 16.0, 16.0, 16.0, 16.0
+    contentMargin: 16.0, 16.0, 16.0, 16.0
```

Comment:
put sprite changes in an abstract prototype they all inherit from
also you dont have to change background margin etc just to change colour